docs/content/routing/providers/kubernetes-crd.md

```diff
@@ -1609,8 +1609,8 @@ or referencing TLS options in the [`IngressRoute`](#kind-ingressroute) / [`Ingre
 
 `TLSStore` is the CRD implementation of a [Traefik "TLS Store"](../../https/tls.md#certificates-stores).
 
-Register the `TLSStore` kind in the Kubernetes cluster before creating `TLSStore` objects
-or referencing TLS stores in the [`IngressRoute`](#kind-ingressroute) / [`IngressRouteTCP`](#kind-ingressroutetcp) objects.
```

Comment:
I think this sentence introduces what the section is about. It would still be accurate to say that it's about referencing a TLS store.

Reply:
I'm not sure to follow you, sorry, maybe I missed something. 
If we can only use one _default_ TLSStore, why a user would be interested in referencing a TLSStore on an IngressRoute ?

Reply:
Yes, sorry the selection targeted was too large for my comment to make sense.
I think that the second part of the sentence (`referencing TLS stores in ....`), was to introduce what the section is about. It is the same wording as in other sections for other resources on this page.
The main issue with that part of the sentence is that it is only allowed to reference the "default" `tlsStore`, and hence we are here documenting something that does not need to be configured/referenced (as the "default" `tlsStore` is always used anyway).